src/main/java/org/folio/inventory/dataimport/handlers/actions/ReplaceInstanceEventHandler.java

```diff
@@ -252,8 +283,8 @@ private void getActualInstanceAndReInvokeCurrentHandler(Instance instance, Insta
       })
       .exceptionally(e -> {
         eventPayload.getContext().remove(CURRENT_RETRY_NUMBER);
-        LOGGER.error(format("Cannot get actual Instance by id: %s", e.getCause()));
-        promise.fail(format("Cannot get actual Instance by id: %s", e.getCause()));
+        LOGGER.error("Cannot get actual Instance by id: {}", instance.getId(), e);
```

Comment:
Why this change?

Reply:
the main purpose of the changes is to pass to logger the exception itself instead of its cause, which actually can be a `null`. In addition, logger parametrized message has been applied instead of string formating.